Marlin/temperature.cpp

```diff
@@ -161,6 +161,9 @@ int Temperature::maxttemp_raw[HOTENDS] = ARRAY_BY_HOTENDS(HEATER_0_RAW_HI_TEMP ,
 int Temperature::minttemp[HOTENDS] = { 0 };
 int Temperature::maxttemp[HOTENDS] = ARRAY_BY_HOTENDS1(16383);
 
+int Temperature::consecutive_low_temperature_error[HOTENDS] = ARRAY_BY_HOTENDS(0, 0, 0, 0);
+unsigned long Temperature::preheatStartTime[HOTENDS] = ARRAY_BY_HOTENDS(0, 0, 0, 0);
```

Comment:
For arrays initialized to zero, you can just use `{ 0 }` rather than these macros.